torch/csrc/jit/script/ir_emitter.cpp

```diff
@@ -2481,6 +2485,13 @@ struct to_ir {
         auto result = emitIsInstance(apply.inputs()[0], apply.inputs()[1]);
         return std::make_shared<SimpleValue>(result.value());
       }
+      case prim::ToList: {
+        auto select = Select(apply.callee());
+        auto value = select.value();
+        auto operand = emitSugaredExpr(value, 1);
```

Comment:
Can you add error handling for when no `type_hint` is passed in, ideally with a helpful error message ? Most users will be coming from python eager where they didn't have a type hint, so pretty much everyone who uses `tolist` will see this error message.

Reply:
Done, although I'm not sure how helpful the message is? I wasn't sure how to get the expected type annotation at this point.